### ISSUE TITLE
Solution to loss explosion

### ISSUE THREAD

**Issue by fkeufss** [TomTomTommi/HiNet#13]
Thank you for sharing your code. I am trying your code and I do find the loss explosion problem. Do you know the inherent reason of it? Is there any better solution instead of restarting training with lower learning rate every time manually?

**Comment by TomTomTommi** [comment]
Hi, thanks for your interest. Actually, this problem occurs frequently and deserves further study, but I have not analyzed it.

**Comment by Hatermelon** [comment]
> Thank you for sharing your code. I am trying your code and I do find the loss explosion problem. Do you know the inherent reason of it? Is there any better solution instead of restarting training with lower learning rate every time manually?

Hello, can you continue the training normally after modifying the parameters manually? I am using the manual method to modify the loss explosion problem for the first time, why after modifying the learning rate and other parameters according to the method, the model re the first round started and did not continue for 500 epochs, the learning rate did not change according to the modifications, is it something I have overlooked? Thank you.

**Comment by lyq2335458686** [comment]
Hello, when I run your code, I obviously downloaded CUDA, but why can't I call the GPU when running?
